### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by davidbeechey**
Updated README to include more info...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,5 +5,121 @@
...
+- [STM32L476RG](https://www.st.com/en/microcontrollers-microprocessors/stm32l476rg.html)
+- [STM32F767ZI](https://www.st.com/en/microcontrollers-microprocessors/stm32f767zi.html)
+- [STM32H743](https://www.st.com/en/microcontrollers-microprocessors/stm32h743zi.html)
+- [STM23L432](https://www.st.com/en/microcontrollers-microprocessors/stm32l432kc.html)
+
+| Sub-system         | Microcontroller | Number |
+| ------------------ | --------------- | ------ |
+| Localisation       | STM32H743       | 1      |
+| Telemetry          | STM32H743       | 1      |
+| Levitation control | STM32F767       | 8      |
```
Comment (TomLonergan03) [inline README.md]: Won't this be 6? or are you controlling opposing horizontal lev units independently?

Comment (davidbeechey) [reply README.md]: honestly not sure, will leave it TBD for now

**Review by arjunnaha — APPROVED** [review]

**Review by jpfbastos — APPROVED** [review]
Looks nice!

**Review by licornes-fluos — APPROVED** [review]

**Review by misha7b — APPROVED** [review]
LGTM!

**Review by kshxtij — APPROVED** [review]
LGTM

**Pull request merged**